datafusion/expr/src/planner.rs

```diff
@@ -49,11 +54,19 @@ pub trait ContextProvider {
         not_impl_err!("Table Functions are not supported")
     }
 
-    /// This provides a worktable (an intermediate table that is used to store the results of a CTE during execution)
-    /// We don't directly implement this in the logical plan's ['SqlToRel`]
-    /// because the sql code needs access to a table that contains execution-related types that can't be a direct dependency
-    /// of the sql crate (namely, the `CteWorktable`).
+    /// Provides an intermediate table that is used to store the results of a CTE during execution)
+    ///
+    /// CTE stands for "Common Table Expression"
+    ///
+    /// # Notes
+    /// We don't directly implement this in ['SqlToRel`] as implementing this function
```

Comment:
This paragraph is not well formatted.

![CleanShot 2025-02-15 at 22 07 48@2x](https://github.com/user-attachments/assets/d49f39d8-faf7-4dae-98a3-90ad845d74d8)

Reply:
fixed in 54fac30cd

![Uploading Screenshot 2025-02-16 at 6.44.37 AM.png…]()